stats/opentelemetry/internal/tracing/custom_carrier.go

```diff
@@ -26,16 +26,28 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+// PropagationDirection specifies whether the propagation is incoming or
+// outgoing.
+type PropagationDirection int
```

Comment:
Can we unexport this and have NewIncomingCarrier and NewOutgoingCarrier?

Reply:
Done